users/signals.py

```diff
@@ -3,10 +3,9 @@
 from django.contrib.auth.models import User
 from django.core.mail import send_mail
 from django.conf import settings
-from django.contrib import messages
 
 
-def create_profile(sender, instance, created, **kwargs):
+def create_profile(instance: Profile, created: bool, **kwargs):
```

Comment:
Warto też otypować, co ta funkcja zwraca :grinning: 

Jeżeli nic, to ` -> None`.